### DISCUSSION TITLE
Уведомления за несколько минут до начала пары

### DISCUSSION THREAD

**Discussion by 0niel** [0niel/university-app#216] [Идеи]
**Проблема.**
Во время дистанционных лекций, сидя за ПК часто забываешь о том, что лекция скоро начнётся. Во время очных пар и большой перемены иногда хочется чиллить и не смотреть телефон.

**Решение**
Настраиваемые локальные Push уведомления за некоторое время до начала пары.

**Comment by d3adwolf** [comment]
Может просто сделать интеграцию в Гугл календарь, а уже в нём гибко настроить оповещения?

**Reply by 0niel** [reply]
Готово!

**Reply by d3adwolf** [reply]
Спасибо, но я уже получил диплом. :)
